sdc-plugin/sdc.cc

```diff
@@ -52,11 +55,16 @@ struct ReadSdcCmd : public Frontend {
         log("\nReading clock constraints file(SDC)\n\n");
         size_t argidx = 1;
         extra_args(f, filename, args, argidx);
-        std::string content{std::istreambuf_iterator<char>(*f), std::istreambuf_iterator<char>()};
-        log("%s\n", content.c_str());
+        file_name = filename;
+        line_number = 0;
         Tcl_Interp *interp = yosys_get_tcl_interp();
-        if (Tcl_EvalFile(interp, args[argidx].c_str()) != TCL_OK) {
-            log_cmd_error("TCL interpreter returned an error: %s\n", Tcl_GetStringResult(interp));
+        while (!f->eof()) {
+            std::string line;
+            std::getline(*f, line);
+            line_number++;
+            if (Tcl_Eval(interp, line.c_str()) != TCL_OK) {
+                log_cmd_error("TCL interpreter returned an error: %s\n", Tcl_GetStringResult(interp));
```

Comment:
This should probably be `log_file_error` too, right?

Reply:
Yes, that is correct. Fixed.